src/native/hash/bitfn.h

```diff
@@ -22,8 +22,7 @@
  * THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-#ifndef BITFN_H
-#define BITFN_H
+#pragma once
```

Comment:
According to the wikipedia article on [Pragma once](https://en.wikipedia.org/wiki/Pragma_once) revision dated 2016-01-13 18:53 last written by 82.19.135.189, this is not supported by Solaris Studio C/C++. Given this fact, I believe that the portability vs readability tradeoffs presented by this change more weighs towards the former, especially since the world at present has a lack of well-written security code - e.g. c.f. arguments favouring the use of LGPL over GPL on a few but highly strategic FOSS projects.